docs/prerequisites.rst

```diff
@@ -53,20 +53,20 @@ Install the prerequisites using APT: ::
 
     sudo apt-get install curl g++ git libffi-dev libjpeg-dev libldap2-dev libsasl2-dev libssl-dev make python-dev python-pip zlib1g-dev
 
-MongoDB 2.6 requires a special incantation to install at this time. Install
+MongoDB 3.2 requires a special incantation to install at this time. Install
 the APT key with the following: ::
 
-    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
 
 For Debian, create the following configuration file for the MongoDB APT repository: ::
 
-    echo 'deb http://downloads-distro.mongodb.org/repo/debian-sysvinit dist 10gen' \
-        | sudo tee /etc/apt/sources.list.d/mongodb.list
+    echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.2 main" \
+        | sudo tee /etc/apt/sources.list.d/mongodb-org-3.2.list
 
 For Ubuntu, instead create the following configuration file: ::
 
-    echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' \
-        | sudo tee /etc/apt/sources.list.d/mongodb.list
+    echo "deb http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.2 multiverse" \
```

Comment:
Should you explicitly mention that this is for Ubuntu 16.04 since you reference `xenial`?

Reply:
I can if you'd like. This entire page is woefully inaccurate already though, hence #2222.